etcdmain/config.go

```diff
@@ -184,6 +185,7 @@ func newConfig() *config {
 	// logging
 	fs.BoolVar(&cfg.Debug, "debug", false, "Enable debug-level logging for etcd.")
 	fs.StringVar(&cfg.LogPkgLevels, "log-package-levels", "", "Specify a particular log level for each etcd package (eg: 'etcdmain=CRITICAL,etcdserver=DEBUG').")
+	fs.StringVar(&cfg.logOutput, "log-output", "", "Specify 'stdout' or 'stderr' to skip journald logging even when running under systemd.")
```

Comment:
The systemd/journald stuff seems forced, maybe:
`fs.StringVar(&cfg.logOutput, "log-output", "default", "Specify a logging target (stdout,stderr,default).")`